data/build.gradle

```diff
@@ -98,7 +98,10 @@ dependencies {
   )
   androidTestImplementation('androidx.test:runner:1.2.0',
       'androidx.test.espresso:espresso-core:3.2.0')
-  kapt("com.squareup.moshi:moshi-kotlin-codegen:1.11.0")
+  kapt(
+      "com.squareup.moshi:moshi-kotlin-codegen:1.11.0",
+      "com.google.dagger:dagger-compiler:2.24"
```

Comment:
Nit: the deps should be in the other order for lexicographical sorting.

Reply:
Done.